src/svg/ds6/icons.svg

```diff
@@ -31,7 +31,7 @@
         <path d="M12.186 18.285c-.451-.009-.809-.167-1.075-.441l-9.337-9.6a1.527 1.527 0 0 1-.424-.999v-.108c.015-.386.166-.741.424-1.008.56-.573 1.529-.57 2.082 0l8.294 8.53 8.292-8.532c.558-.57 1.526-.57 2.08 0 .265.27.416.629.428 1.01v.087c-.012.391-.165.75-.427 1.02l-9.333 9.6a1.443 1.443 0 0 1-1.004.441"/>
     </symbol>
     <symbol id="icon-checkbox-unchecked" viewBox="4.12 4.1 15.77 15.8">
-        <path fill-rule="evenodd" d="M18.56 5.25H5.44a.19.19 0 0 0-.19.187v13.126a.19.19 0 0 0 .19.187h13.12a.19.19 0 0 0 .19-.187V5.437a.19.19 0 0 0-.19-.187zM4.116 19.9h15.768V4.1H4.116v15.8z"/>
+        <path fill="#c8c8c8" fill-rule="evenodd" d="M18.56 5.25H5.44a.19.19 0 0 0-.19.187v13.126a.19.19 0 0 0 .19.187h13.12a.19.19 0 0 0 .19-.187V5.437a.19.19 0 0 0-.19-.187zM4.116 19.9h15.768V4.1H4.116v15.8z"/>
```

Comment:
This shouldn't change.

Reply:
It appears this is my fault. Let me fix it.

Reply:
Fixed here: https://github.com/eBay/skin/pull/740/files#diff-0414bda8012ba8a5234866832070991aR34

Reply:
Pushed those changes @seangates, let me know if it's fine now.
Not sure about the windows high contrast one though.